.github/workflows/ccip-offchain-upgrade-tests.yml

```diff
@@ -1,390 +0,0 @@
-name: CCIP Offchain Upgrade Compatibility Tests
```

Comment:
Is there any alternative?

Reply:
For now it's messy. If we fix it to run okay now, it risks testing the wrong version upgrade path and giving us false negatives. If we leave it broken, it risks people disregarding it as a flake in the future. The plan is to revive it [TT-1448] when we've ironed out the new release process for CCIP in the main repo.